testing/smoke-ide-test/build.gradle.kts

```diff
@@ -17,6 +15,17 @@ val smokeIdeTestSourceSet = sourceSets.create("smokeIdeTest") {
 addDependenciesAndConfigurations("smokeIde")
 val smokeIdeTestImplementation: Configuration by configurations
 val smokeIdeTestDistributionRuntimeOnly: Configuration by configurations
+val ideStarter by configurations.creating {
```

Comment:
🤔 Don't we have a better factory method to create these configurations?

Reply:
This pattern is widespreaded in the codebase.
Do you mean you would prefer something less verbose?

Reply:
Less verbose, more idiomatic. I was thinking about the recently introduced `consumable`, `dependencyScope` and `resolvable` factory methods but it seems neither provides what we're looking for here (cc @gradle/bt-jvm).